doc/ddu-filter-converter_devicon.txt

```diff
@@ -93,14 +66,87 @@ defaultIconHlgroup	(string)
 
 			 *ddu-filter-converter_devicon-param-specificFileIcons*
 specificFileIcons	(dictionary)
```

Comment:
`|Dictionary|` is better.
There are several other places, but individual comments will be omitted.